VAMobile/src/screens/PaymentsScreen/DirectDepositScreen/EditDirectDepositScreen/EditDirectDepositScreen.tsx

```diff
@@ -122,6 +122,7 @@ function EditDirectDepositScreen({ navigation, route }: EditDirectDepositProps)
           if (!routingNumberError) {
             snackbar.show(t('directDeposit.saved.error'), {
               isError: true,
+              offset: 0,
```

Comment:
I think Narin had already suggested it somewhere, but also suggesting having the offset for non-Nav screens set in the app theming instead of having so many hard coded 0's. Make it much easier to change in the future if it's desired vs updating ~25 spots.

Reply:
Agree, I will make this change.